azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -492,6 +512,19 @@ def az_cli(self):
     def token_provider(self):
         return self._token_provider
 
+    @property
+    def interactive_login(self):
+        val = self._internal_dict.get(self.ValidKeywords.interactive_login)
+        return val is not None and val

```

Comment:
I'm pretty sure that `bool(val)` would achive the same effect and be more readable

Reply:
You resolved this and previous comment, but I don't see any comment from you or change in the code